otx/algorithms/segmentation/adapters/mmseg/datasets/dataset.py

```diff
@@ -257,12 +261,18 @@ def __init__(self, **kwargs):
         if test_mode is False:
             self.img_indices = get_old_new_img_indices(classes, new_classes, otx_dataset)
 
+        if test_mode:
+            for pipe in pipeline:
+                if pipe["type"] == "LoadFromFiles":
```

Comment:
Actually I don't know what this is for.. "LoadFromFiles", Why especially only for test? Could you explain?

Reply:
For training mode - it is controlled by loading function in pipelines, much simpler to change